docs/UserGuide.md

```diff
@@ -180,6 +182,41 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd person in the address book.
 * `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 
+### Creating an event: `create`
+
+Creates a new event in the address book.
+
+Format: `create t/EVENT_TITLE d/DATE t/TIME p/PURPOSE`
+
+* The compulsory parameters are: event title (t), date (d), time (t) and purpose (p)
```

Comment:
Perhaps you can format the command prefixes as inline code to signal to the reader that they are part of the command?

```suggestion
* The compulsory parameters are: event title (`t`), date (`d`), time (`t`) and purpose (`p`)
```

Reply:
Yes, will do this